### DISCUSSION TITLE
changing cameras

### DISCUSSION THREAD

**Discussion by Aripaule** [aaronwmorris/indi-allsky#1746] [Q&A] [answered]
Hi,

 I'm testing with a new camera, from another brand (zwo, before it was playerone) but it doesn't appear in the selection menu. I already did the update in an attempt to install the driver but it didn't seem to work. Any solution without having to re-install? thanks.

**Comment by aaronwmorris** [comment, marked as answer]
You can either re-run setup.sh and select to update the indiserver (you will be able to select a new driver).  Or there is a dedicated script to change the indiserver, `./misc/camera_change.sh`.  Both will do the same thing.
